tests/target/issue-5507/issue-5507-version-two.rs

```diff
@@ -0,0 +1,35 @@
+// rustfmt-version: Two
+
+struct EmptyBody<T>
+where
+    T: Eq,
+{}
```

Comment:
Subjective, but I think this should stay on the same line as the bound, Essentially following the `version=One` formatting.

Reply:
Okok. I'll proceed to fix it